test/index.js

```diff
@@ -76,4 +81,12 @@ describe('cards filtering', function () {
 		filteredCards.forEach(card => expect(card.back).to.not.be.undefined);
 		filteredCards.forEach(card => expect(card.headingLevel % 2).to.equal(0));
 	});
+	it('should find one image in tokens', function() {
+		let items = 0;
+		images.forEach(image => {
+			items++;
+		});
+
+		expect(items).to.equal(1);
```

Comment:
```suggestion
		expect(images.length).to.equal(1);
```
Ich denke `images.length` tut's auch, oder?

Reply:
Leider nicht, wenn Objekte in einem Array sind kommt da immer 0 raus.
JS ist komisch.

Reply:
MOMENT MAL

Ich schwöre gestern hat das nicht funktioniert...............

Reply:
Wollte schon sagen, bei JS gibt's ja so ein paar Sachen über die man Zweifeln kann, aber das hätte mich wirklich gewundert